libs/shared/src/consts/providers/channels/sms.ts

```diff
@@ -153,4 +154,12 @@ export const smsProviders: IProviderConfig[] = [
     docReference: 'https://developers.termii.com/authentication',
     logoFileName: { light: 'termii.png', dark: 'termii.png' },
   },
+  {
+    id: SmsProviderIdEnum.Pinpoint,
+    displayName: 'Pinpoint',
+    channel: ChannelTypeEnum.SMS,
+    credentials: pinpointConfig,
+    docReference: 'https://docs.aws.amazon.com/pinpoint/index.html',
+    logoFileName: { light: 'termii.png', dark: 'termii.png' },
```

Comment:
We should use the files for the Pinpoint logos and add them to the monorepo, because if not we will confuse the users.